docs/user-guide/tree_structures.ipynb

```diff
@@ -0,0 +1,433 @@
+{
```

Comment:
<p>Slight rephrasing</p><p> </p><p>The <code>BallTree</code> class can be accessed through the <code>Grid.get_ball_tree(coordinates, coordinate_system, distance_metric, reconstruct)</code> method, which takes in the following parameters:</p><p> </p><p>The rest can be unchanged </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/806/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>